noxfile.py

```diff
@@ -1,12 +1,58 @@
+import os
+import sys
+import subprocess
 from pathlib import Path
 
 import nox
-import sys
+
+
+PYODIDE_VERSION = os.getenv("PYODIDE_VERSION", "0.21.0-alpha.2")
+GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS")
+GITHUB_ENV = os.getenv("GITHUB_ENV")
+
+
+def append_to_github_env(name: str, value: str):
+    if not GITHUB_ACTIONS or not GITHUB_ENV:
+        return
+
+    with open(GITHUB_ENV, "w+") as f:
+        f.write(f"{name}={value}\n")
+
+
+@nox.session(name="setup-pyodide")
+def setup_pyodide(session: nox.Session):
+    tests_dir = Path("./tests").resolve()
+    with session.chdir(tests_dir):
+        session.run(
+            "npm",
+            "i",
+            "--no-save",
+            f"pyodide@{PYODIDE_VERSION}",
```

Comment:
Maybe pass the Pyodide version as an argument to the session rather than as an environment variable for easier command line use?